.github/workflows/builder-dispatch.yml

```
@@ -0,0 +1,38 @@
+---
+name: Manual dispatch of package builds
+
+on:
+  workflow_dispatch:
+    inputs:
+      product:
+        description: Product to build
+        type: choice
+        options:
+        - authoritative
+        - recursor
+        - dnsdist
+      os:
+        description: OS to build for
+        type: string
+
+jobs:
+  build:
+    name: build ${{ github.event.inputs.product }} for ${{ github.event.inputs.os }}
+    # on a ubuntu-20.04 VM
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2.3.4
+        with:
+          fetch-depth: 0  # for correct version numbers
+          submodules: recursive
+      # this builds packages and runs our unit test (make check)
```

Not sure if you imagine having more than one unit test.
```suggestion
      # this builds packages and runs our unit tests (make check)
```